### PR TITLE
Created blue btn in home page

### PR DESCRIPTION
**Pull request by yannklein**
Check the new button, leading to the About page.
image

### CODE REVIEW

**Comment by yannklein** [comment]
Love it! check it out @ashishrajshrestha

**Comment by SSatoka** [comment]
nice

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -4,3 +4,4 @@
 <% end %>
 
 
+<%= link_to "About page", about_path, class: "btn btn-primary" %>
```
Comment (yannklein) [inline app/views/pages/home.html.erb]: Can you put some BS container and row around your btn?

**Comment by ashishrajshrestha** [comment]
can u change the font type.. It might look better.

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (yannklein) [reply app/views/pages/home.html.erb]: Done

**Review by yannklein — COMMENTED** [review]
Done

**Pull request merged**